internal/controller/util/misc.go

```diff
@@ -22,6 +22,9 @@ const (
 	OCMBackupLabelValue string = "ramen"
 
 	IsCGEnabledAnnotation = "drplacementcontrol.ramendr.openshift.io/is-cg-enabled"
+
+	// Annotation
```

Comment:
The comment is not useful - maybe describe here how this annotation change the system behavior?

Reply:
Indeed.